src/components/Highlight.vue

```diff
@@ -10,6 +10,13 @@ import { createSuitMixin } from '../mixins/suit';
 import { getPropertyByPath } from '../util/object';
 import { warn } from '../util/warn';
 
+// not imported from instantsearch.js/es/lib/escape-highlight,
+// because that path can change adn break our implementation
+const TAG_REPLACEMENT = {
```

Comment:
@francoischalifour WDYT about this duplication instead of import?

Reply:
We can inline `mark` where we used `em` because we don't use this anywhere else.